### PR TITLE
Merge front-end to main: Added more documentations, and add status feature.

### PR DESCRIPTION
**Pull request by slok1368**

### CODE REVIEW

File: src/front-end/link-front-and-back/LinkFrontAndBack.cpp [hunk 1]
```diff
@@ -29,37 +26,31 @@ void  __stdcall GetGame(int* id, char* name, char* path, char* imagePath)
 
 	const char* image_src = (*it)->getImagePath();
 	strcpy_s(imagePath, strlen(image_src) + 1, image_src);
-
-	cout << "Successfuly copied: " << (*it)->getName() << endl;
```
Comment (asagrekin) [inline src/front-end/link-front-and-back/LinkFrontAndBack.cpp:33]: Good job cleaning up code.

File: src/front-end/front-end-interface/MainScreen.cs [hunk 2]
```diff
@@ -134,21 +173,24 @@ private void EditGameButton_Click(object sender, EventArgs e)
...
         {
             this.gamesScrollPanel.Size = new Size(this.gamesList.Width, ClientSize.Height - 2 * this.logo.Height);
             this.editScrollPanel.Size = new Size(this.gamesEditList.Width, ClientSize.Height - 2 * this.logo.Height);
         }
 
+        // When `Save Order` is pressed, delete and append each game to 
+        // save the new order to database.
         private void SaveOrderButton_Click(object sender, EventArgs e)
         {
-            Console.WriteLine("New Order:");
```
Comment (asagrekin) [inline src/front-end/front-end-interface/MainScreen.cs:145]: I like the name changes.

**Review by asagrekin — APPROVED** [review]
Looks good for merging to me.

**Review by lochoa2 — COMMENTED** [review]
Good comments, and style.

**Pull request merged**